database/tables.sql

```diff
@@ -8,7 +8,7 @@ CREATE TABLE "Orders" (
   "customerId" INT NOT NULL,
   "orderDate" date DEFAULT CURRENT_DATE,
   "totalPrice" float NOT NULL DEFAULT 1,
-  "orderItems" jsonb[] NOT NULL DEFAULT '{}',
```

Comment:
This has to be an array of orders so that we can argue that we have object oriented features, we currently push hoodies info as documents to this array, an array of documents